.github/actions/javascript/proposalPoliceComment/proposalPoliceComment.ts

```diff
@@ -3,15 +3,12 @@ import {context} from '@actions/github';
 import type {IssueCommentCreatedEvent, IssueCommentEditedEvent, IssueCommentEvent} from '@octokit/webhooks-types';
 import {format} from 'date-fns';
 import {toZonedTime} from 'date-fns-tz';
+import {convertToNumber} from '@github/libs/ActionUtils';
 import CONST from '@github/libs/CONST';
 import GithubUtils from '@github/libs/GithubUtils';
-import sanitizeJSONStringValues from '@github/libs/sanitizeJSONStringValues';
-import OpenAIUtils from '../../../../scripts/utils/OpenAIUtils';
-
-type AssistantResponse = {
-    action: typeof CONST.NO_ACTION | typeof CONST.ACTION_REQUIRED;
-    message: string;
-};
+import ProposalPoliceTemplates from '@prompts/proposalPolice';
+import OpenAIUtils from '@scripts/utils/OpenAIUtils';
+import type {AssistantResponse, DuplicateProposalResponse} from '@scripts/utils/OpenAIUtils';
```

Comment:
These types do not belong in `OpenAIUtils` because they're specific to ProposalPolice.